src/runtime/components/ScriptGoogleAdsense.vue

```diff
@@ -37,9 +36,7 @@ const instance = useScriptGoogleAdsense({
 const { status } = instance
 
 onMounted(() => {
-  callOnce(() => {
```

Comment:
I did do this for a reason :thinking: 

Possibly because in dev the HMR was triggering a new ad each time. 

We might need to track which ads we've pushed an object for if we're going to remove this